app/models/partner_benefit_model.rb

```diff
@@ -0,0 +1 @@
+class PartnerBenefitModel < BenefitModel; end
```

Comment:
Is there some way of avoiding this class needing to exist? If there is no difference in behaviour..?

Reply:
As discussed, the benefit of this class is it having its own i18n key for error messages. I have added a comment to this file to that effect.